internal/cli/command/create/create.go

```diff
@@ -44,7 +45,7 @@ func NewCmd(octaviusDaemon daemon.Client, fileUtil file.File) *cobra.Command {
 		},
 	}
 	createCmd.Flags().StringVarP(&metadataFilePath, "job-path", "", "", "path to metadata.json(required)")
-	createCmd.MarkFlagRequired("job-path")
+	_ = createCmd.MarkFlagRequired("job-path")
```

Comment:
it will be better if we not ignore the error returned and rather treat it the way it should be. At all the places.